code/API_definitions/EdgeCloud_LcM.yaml

```diff
@@ -112,60 +108,38 @@ paths:
               example:
                 status: 409
                 code: CONFLICT
-                message: "Application already instantiated in the given Edge or Region"
+                message: "App duplicated"
```

Comment:
Agree with you. Changed!

Reply:
I would add we should decide (and document) whether given names will be treated as unique identifiers or not.

Reply:
Perfect. We will add in the documentation which given names will be treated as unique identifiers.

I'm not sure if the criteria to declare that an `application already exists` is only the `name`, I mean that the criteria should be `name` and `version` because developers can have the same application name but different versions.

Reply:
Yes, we use the pair `name`+`version` as the unique identifier. It would be easiest for us to continue with that approach, but we are open to discussion on this.

Reply:
@gunjald, @gainsley. I think creating a new Discussion on this topic would help in deciding how to define unique identifiers. Could one of you create one?